### PR TITLE
Create Dash 2 stub package

### PR DESCRIPTION
**Pull request by jonmmease**
Stub package that Dash 2 will depend on to provide a `dash_core_components` import that is backward compatible with Dash 1. Raises a warning on import.  Stub package has version 2.0.0rc1.

The diff is not very helpful. Probably easiest to check out the branch and look at what's left

### CODE REVIEW

**Comment by jonmmease** [comment]
ping @alexcjohnson  :woman_dancing: ?

File: MANIFEST.in [hunk 1]
```diff
@@ -1,7 +1,2 @@
-include dash_core_components/*.js
-include dash_core_components/*.js.map
-include dash_core_components/metadata.json
-include dash_core_components/package-info.json
 include README.md
 include LICENSE.txt
```
Comment (alexcjohnson) [inline MANIFEST.in]: We have a `LICENSE`, not `LICENSE.txt`... yet the rc1 tarball contains the file anyway, is that just included by default?

**Review by alexcjohnson — APPROVED** [review]
💃

File: MANIFEST.in (hunk 1 above)
Comment (jonmmease) [reply MANIFEST.in]: huh, not sure what rule caused this to work anyway. But I removed the extension from MANIFEST.in in 6928a75

**Pull request merged**